source/file/file_test.go

```diff
@@ -16,7 +16,9 @@ func Test(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.RemoveAll(tmpDir)
+	defer func() {
+		_ = os.RemoveAll(tmpDir)
```

Comment:
We can use `t.Error()` or `t.Fatal()` here (and in the rest of this file) as well.